### PR TITLE
Upgrade Warsow to 2.0

### PR DESCRIPTION
**Pull request by lieuwex**

### CODE REVIEW

File: Casks/warsow.rb [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-  url "http://update.warsow.net/mirror/warsow_#{version}.dmg.zip"
+  url "https://www.warsow.gg/download?dl=warsow#{version.delete('.')}mac"
   name 'Warsow'
   homepage 'http://www.warsow.net/'
   license :gpl
 
-  container :nested => "Warsow #{version}.dmg"
-
-  app "Warsow #{version}.app"
+  app "Warsow.app"
```
Comment (vitorgalvao) [inline Casks/warsow.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by vitorgalvao** [comment]
Have you tested the installation after the changes, to see if it doesn’t choke and correctly identifies the container?

**Comment by lieuwex** [comment]
@victorpopkov Yeah, I tried installing it and everything seems to be okay.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 434fb79d6efd136c0e70aaec4939d6c17f7e2db7 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**